### PR TITLE
migrate to test^0.12.0

### PR DESCRIPTION
**Pull request by a14n**
To use with `pub run test -p dartium -p chrome -p firefox`.

### CODE REVIEW

File: test/notification_test.dart [hunk 1]
```diff
@@ -61,8 +61,8 @@ main() {
 }
 
 /// Simulate clicking on the notification.
-clickOn(thing) {
+clickOn(Notification thing) {
   var event =
-      new MouseEvent('click', view: thing, canBubble: true, cancelable: true);
+      new MouseEvent('click', canBubble: true, cancelable: true);
```
Comment (a14n) [inline test/notification_test.dart]: Oops, need to format the code.

Comment (a14n) [inline test/notification_test.dart]: updated

**Comment by alan-knight** [comment]
Sorry, missed the notification on this. I think it's superceded now by the other test-related pull request.

**Comment by a14n** [comment]
no problem :)

**Pull request closed without merging**